drv/lpc55-update-server/src/main.rs

```diff
@@ -152,7 +159,7 @@ impl idl::InOrderUpdateImpl for ServerImpl {
             flash_page[len..].fill(0);
         }
 
-        do_block_write(target, block_num, &mut flash_page)?;
+        do_block_write(&mut self.flash, target, block_num, &mut flash_page)?;
```

Comment:
```suggestion
        do_block_write(&mut self.flash, target, block_num, &flash_page)?;
```

Reply:
Ha, thanks -- `do_block_write` used to take a `&mut`, must have missed this one. Changed.